HIP-xxxx.md

```diff
@@ -0,0 +1,163 @@
+# HIP-xxxx : Standard for TXT records
+
+```
+Number:  HIP-xxxx
+Title:   Standard for TXT records
+Type:    Standards
+Status:  Draft
+Authors: 0xStefan <https://twitter.com/0xStefan>
+Created: 2022-02-22
+```
+
+## Abstract
+
+A proposal for TXT record standards to provide on-chain data for a domain.
+
+## Motivation
+
+#### Domain Parking
+
+There's currently no way to know if a domain is for sale by the owner or not.
+A domain seller has no reliable way to provide that information.
+
+#### Identity / Web3 Profile
+
+Websites using Sign in with Handshake can use and render the TXT record data, e.g. as profile information. The domain owner is still in control of the data.
+
+## TXT Standards Definition
+
+For the above to work we need standards so any website can properly render the TXT records. The records should be flexible, short and easy to understand. Since it's impossible to validate the data via HSD. Therefore TXT records should be provided mostly without any prefix like "mailto:", "tel:" or "https://www.twitter.com/".
+
+The goal is to let the app or website decide how to render the information. It also decreases the data saved on-chain.
+
+#### Domain Parking
+
+The parking TXT entry indicates that the domain is for sale.
+
+```
+TXT parking=[String]
+```
+
+#### Contact or Profile Information
+
+```
+TXT email=[Email]
+TXT phone=[Tel]
+```
+
+```
+TXT discord=[String]
+TXT facebook=[String]
+TXT github=[String]
+TXT instagram=[String]
+TXT linkedin=[String]
+TXT telegram=[String]
+TXT tiktok=[String]
+TXT twitter=[String]
+TXT weixin=[String]
+TXT whatsapp=[Tel]
+TXT youtube=[String]
```

Comment:
Is this list static?
If so, this seems opinionated.
If not, the list will be unmaintainable.

Reply:
After thinking about this, I agree that it will be impossible to maintain such a list.
Therefore I will propose a standard in a new HIP that is standardized but flexible enough to enable that. Closing this so we can discuss it further (if needed) in a new PR.